index.js

```diff
@@ -173,20 +189,30 @@ function html (elementType, attributes, children) {
   return element
 }
 
+function determineNumberOfProposals(proposals) {
+  proposals.forEach(function (proposal) {
+    states[proposal.status.state].count = states[proposal.status.state].count + 1;
+  });
+
+  // .acceptedWithRevisions proposals are combined in the filtering UI
+  // with .accepted proposals.
+  states['.accepted'].count = states['.accepted'].count + states['.acceptedWithRevisions'].count
+}
```

Comment:
Can you use the [`+=`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Assignment_Operators#Addition_assignment) operator here?

```JavaScript
function determineNumberOfProposals (proposals) {
  proposals.forEach(function (proposal) {
    states[proposal.status.state].count += 1
  })

  // .acceptedWithRevisions proposals are combined in the filtering UI
  // with .accepted proposals.
  states['.accepted'].count += states['.acceptedWithRevisions'].count
}
```

Reply:
@benrimmington It's great, thank's